tests/foreman/ui/test_discoveryrule.py

```diff
@@ -154,7 +154,7 @@ def test_negative_delete_rule_with_non_admin_user(
         with pytest.raises(ValueError):  # noqa: PT011 - TODO Adarsh determine better exception
             session.discoveryrule.delete(dr.name)
         dr_val = session.discoveryrule.read_all()
-        assert dr.name in [rule['Name'] for rule in dr_val]
+        assert dr.name not in dr_val
```

Comment:
```suggestion
        assert dr.name in dr_val
```
@amolpati30 You have changed the assertion here. Is this expected?